### DISCUSSION TITLE
Hosting on Deta Space

### DISCUSSION THREAD

**Discussion by katriellucas** [pocketbase/pocketbase#1767] [Q&A]
Would it be possible to host PB on [Deta Space](https://deta.space/)? I doesn't support docker but it does support Go, so technically, I think so? If anyone have tried, please, let me know.

**Comment by silenceofthewans** [comment]
The most basic pocketbase setup only needs a VPS with persistent storage and open port(s) to work ([_see production guide_](https://pocketbase.io/docs/going-to-production/)). Once you compile your backend/app into a go binary (if extending pocketbase) or if using the pre-compiled binary from pocketbase.io itself, you don't then need go installed on the system to execute it and start the server. 

I looked into deta space and through all the marketing I found this [video](https://www.youtube.com/watch?v=423kzQykSW0). It seems that they want a "Nextcloud" + "Playstore" like experience for web where you will push you app to their "Personal Cloud", at this stage it's available only to you and is for development but after that you "release" it and it gives you a URL which you provide to your end users and they "install" it on their own personal cloud. Also, all the users instances are isolated from other instances. So, in a nutshell it's like running a seperate pocketbase instance for every single user. If each user of your app is isolated (like a non-collaborative notes app etc) then it should work.

At the moment of writing this, pocketbase leans heavily towards a single server setup (vertical scaling) due to sqlite and a focus on simplicity and portability, and also because pb's hooks (which are important if you are extending it with go) are not designed to trigger across seperate instances aka horizontal scaling.
